docs/developer-guide/developing.md

```diff
@@ -125,6 +125,7 @@ The back end will run on port 8080 and will look for the front-end at port 8081.
 
 ```sh
 export MAPSTORE_BACKEND_PORT=8082
+export MAPSTORE_DATA_DIR=/usr/datadir # set the datadir location
```

Comment:
````suggestion
Optionally you can set the data dir location by setting the environment variable `MAPSTORE_DATA_DIR`. 

```sh
export MAPSTORE_BACKEND_PORT=8082 # set a different backend port
export MAPSTORE_DATA_DIR=/usr/datadir # set the datadir location
````